internal/olm/operator/registry/operator_installer.go

```diff
@@ -109,12 +106,11 @@ func (o OperatorInstaller) waitForCatalogSource(ctx context.Context, cs *v1alpha
 			return false, err
 		}
 		if cs.Status.GRPCConnectionState != nil {
-			fmt.Println("grpc connection state:", cs.Status.GRPCConnectionState.LastObservedState)
 			if cs.Status.GRPCConnectionState.LastObservedState == "READY" {
 				return true, nil
 			}
 		} else {
-			fmt.Println("grpc connection state: <nil>")
+			fmt.Println("catalog source connection state: <nil>")
```

Comment:
Is this debug print code? If so, should we remove it?

Also remove `//nolint:unused` from the top of the function now that we're using it.